lib/Backends/CPU/LLVMIRGen.cpp

```diff
@@ -1863,7 +1863,7 @@ void LLVMIRGen::generateLLVMIRForInstr(llvm::IRBuilder<> &builder,
     break;
   }
 
-    // Alloc and Dealloc instructions are handled by the memory allocator.
+  // Alloc and Dealloc instructions are handled by the memory allocator.
```

Comment:
this seems reasonable, but could you extract that into separate commit? Just make sure that your changes are clang formatted.

Reply:
I understand that, but these files do not belong to the PR changes.